api/grpc_server.go

```diff
@@ -446,3 +449,19 @@ func (s SpacemeshGrpcService) GetStateRoot(ctx context.Context, empty *empty.Emp
 	log.Info("GRPC GetStateRoot msg")
 	return &pb.SimpleMessage{Value: s.Tx.GetStateRoot().String()}, nil
 }
+
+func (s SpacemeshGrpcService) AcquirePort(ctx context.Context, in *pb.AcquirePortRequest) (*pb.SimpleMessage, error) {
```

Comment:
I'm not sure if this is really useful. we don't want to use the node as a UPnP proxy. if these API endpoints have a right to exist is because they are taking care of configuring the node port and UPnP, just doing UPnP is kinda weird and not adding any value.

Reply:
I still think this should go down for now..

Reply:
removed